target/linux/ramips/dts/flash_kroks_16m.dtsi

```diff
@@ -0,0 +1,12 @@
+&partitions {
```

Comment:
This PR only adds one device, so please just create one DTS file with all the nodes put together. We may split up stuff when we actually have more devices.

Reply:
done